docs/platform/concepts/maintenance-window.rst

```diff
@@ -17,3 +17,11 @@ In case of **Apache Kafka®** and **OpenSearch®** the service DNS address resol
 .. Note:: 
 
     While the DNS name remains the same, the IP address it points to, will change during a maintenance break. To know more about static IP addresses, check the :doc:`related documentation <static-ips>`.
+
+
+Maintenance updates
+~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Suggested rewording: 

Security updates, platform updates that affect reliability or stability of the service nodes, and quarterly patch updates are always mandatory. Other updates are initially optional. Advance notice is given for all updates. After optional updates have been available for six months, they become mandatory and are applied on the next week's maintenance window at the earliest. This means you have at least 7 days advance notice with exception of critical security updates. These critical updates are applied in the maintenance window of the current week.

Reply:
Updated, thank you so much!